src/Cake.Common/IO/Paths/ConvertableDirectoryPath.cs

```diff
@@ -127,6 +127,86 @@ internal ConvertableDirectoryPath(DirectoryPath path)
         }
 
         /// <summary>
+        /// Operator that combines A <see cref="ConvertableDirectoryPath"/> instance
```

Comment:
Don't need the capital `A` here, should simply be `a`.

Reply:
@grep Well, that was embarrassing. 😳
I noticed the same typo in the XML documentation for the `+` operator, so I fixed it in 98f74d7.